### DISCUSSION TITLE
Channel selection in "audio only mode" (Resolution=0)

### DISCUSSION THREAD

**Discussion by RocketAlf** [GerryDazoo/Slinger#476] [Q&A]
I have made the experience that the channel selection via remote.txt and flask in "audio only mode" (Resolution=0) does not work. But if the selbs Slingbox (M1) is streamed with video (Resolution=5), then the channel selection works.

Is this impression correct and is that intended? It would be nice if the channel selection would also work in "audio only mode".

[SLINGBOXES]
sb1=M1
sb2=M1audio 

[M1]
sbtype="350/500/M1"
password=*******************
ipaddress=192.168.2.175
port=5301
Resolution=5
FrameRate=30
VideoBandwidth=2000
VideoSmoothness=63
VideoSource=0
Remote=dream_remote.txt
 
[M1audio]
sbtype="350/500/M1"
password=*******************
ipaddress=192.168.2.175
port=5301
Resolution=0
FrameRate=30
VideoBandwidth=2000
VideoSmoothness=63
VideoSource=0
Remote=dream_remote.txt
 
Another question. Does the order of declarations (in section [SLINGBOXES]) and the order of definitions matter? So would it work like that?

[SLINGBOXES]
sb1=M1audio
sb2=M1 

[M1]
sbtype="350/500/M1"
....

[M1audio]
sbtype="350/500/M1"
...

Btw ...another question. Has anyone managed to bring the bricked Slingbox SOLO back to life?

**Comment by GerryDazoo** [comment]
1) No the order doesn't matter
2) All bricked Solos remain bricked.

Next time when asking for help please also include ALL the server output. We don't even know for sure what version of the software you're running. And a complete copy of your config.ini. It's OK to redact the slingbox passwords.
Also a more complete description of "Doesn't Work" would be helpful.

**Reply by RocketAlf** [reply]
First of all thank you. I'll be back soon (it's already late in Germany ;-) )

**Reply by RocketAlf** [reply]
@Gerry, I'm back again :-)

What did i do? I stopped Slinger and then cleared the log file. Then I started Slinger again.

First I started the connection to the M1 Slingbox in "audio only mode ([M1audio], Resolution=0) and then tried to change the channel (to channel 44). The channel switch didn't work.
I cut the connection.

Second, I started the connection to the M1 Slingbox in "video mode" ([M1], Resolution=5) and then tried to change the channel (to channel 44). The toggle switch worked this time.
I disconnected again.

All this is recorded in the attached log file. I hope it helps you to find the error.

FYI...Slinger is installed on a Raspi and I'm streaming with VLC on an Android phone. I call up the remote control using the Google Chrome browser. The ProHD and M1 both are connected to a Dreambox which shall be controlled using "dream_remote.txt", M1tv is connected to a tv and not able to control the tv. 

[config.txt](https://github.com/GerryDazoo/Slinger/files/10356436/config.txt)
[sling.log](https://github.com/GerryDazoo/Slinger/files/10356437/sling.log)
[dream_remote.txt](https://github.com/GerryDazoo/Slinger/files/10356463/dream_remote.txt)

**Reply by GerryDazoo** [reply]
There's some things in the logs that are odd, Firstly this
M1audio Stream started at 01/05/2023, 23:49:04.552  578 0
..01/05/2023, 23:49:29.138   RemoteControl connection from ('192.168.2.109', 45740)

Starting remote control stream handler for  ('192.168.2.109', 45740) to port 9091
Remote Control Connected
GET Remote/M1
Exiting Remote Control Stream Handler for ('192.168.2.109', 45740)
.01/05/2023, 23:49:46.636   RemoteControl connection from ('192.168.2.109', 45752)

Starting remote control stream handler for  ('192.168.2.109', 45752) to port 9091
Remote Control Connected
Key Data ('Channel', 'Channel')
Key Data ('Digits', '44')
Sending Channel Digits 44

You stated an M1Audio stream but then tried to channel the channel to 44 using the M1 remote. That will NOT work.

Then you stopped the M1Audio stream
.01/05/2023, 23:50:21.614  M1audio Stream Terminated for  192.168.2.109:45734
Then you started up the M1 stream and changed the channel successfully this time because you were using the correct remote web page. IR [b'\x0c192.168.2.109', b'\x0c192.168.2.109']
And finally about 25 seconds later you closed the stream
...01/05/2023, 23:51:42.863  M1 Stream Terminated for  192.168.2.109:45770

Please remember the Slingbox only supports one stream at a time. If you have a "M1" running starting an "M1Audio" will kill the "M1" stream.

To get back to your original report. The channel isn't changing because you're using the wrong Remote webpage.  Use http://serverip:9090/Remote/M1audio

**Reply by RocketAlf** [reply]
Thank you and sorry for the stupid mistake on my part. I thought that it does not matter if the same device is addressed.
Now of course it works :-)
